### PR TITLE
Continuous aggregate doc update

### PR DESCRIPTION
**Pull request by gayyappan**
1. Support for multiple continuous aggregates.
2. continuous_aggregate_stats view changes

### CODE REVIEW

**Comment by solugebefola** [comment]
Label, please

File: using-timescaledb/continuous-aggregates.md [hunk 1]
```diff
@@ -259,12 +259,14 @@ SELECT min_time::timestamp FROM device_summary;
...
 ### Future Work [](future-work)
-The first version of continuous aggregations has been released in TimescaleDB
-v1.3, but we have a number of new capabilities and improvements already planned
-in the next releases. Please find some of these forthcoming capabilities
-described below. If you'd like to help implement them, test them, want to help
-us prioritize, or have other areas you think we should work on, please get in
-touch via our [Github][timescale-github] or [Slack][support-slack].
+The first version of continuous aggregations was released in TimescaleDB
+v1.3, and we have a number of new capabilities and improvements already planned
+in the next releases. We support the ability to create multiple continuous
```
Comment (solugebefola) [inline using-timescaledb/continuous-aggregates.md]: > We support the ability to create multiple continuous aggregates on a hypertable in TimescaleDB v1.4.

Shouldn't this line come before 

> and we have a number of new capabilities and improvements already planned in the next releases.

?

"Support for creating multiple aggregates on a hypertable was released in v1.4."?

**Comment by solugebefola** [comment]
Just the one comment, otherwise lgtm

File: using-timescaledb/continuous-aggregates.md (hunk 1 above)
Comment (gayyappan) [reply using-timescaledb/continuous-aggregates.md]: modified.

**Pull request merged**